vue-frontend/src/config.js

```diff
@@ -13,6 +13,7 @@ export default Object.freeze({
   API_BASE: "https://dev-stack-api.canopas.com",
   BASE_URL: "https://dev-stack.canopas.com",
   CALENDLY_IFRAME_URL: "https://calendly.com/darpanvithani/15min",
+  CLUTCH_URL: "https://clutch.co/profile/canopas-software#reviews",
```

Comment:
Let's add this url in config-prod also, As we are using it as config in production